elyra/cli/pipeline_app.py

```diff
@@ -258,5 +258,61 @@ def run(pipeline_path):
     print_banner("Elyra Pipeline Local Run Complete")
 
 
+@click.command()
+@click.option('--json',
+              'json_option',
+              is_flag=True,
+              required=False,
+              help='Returns pipeline file in a machine-readable JSON format')
+@click.argument('pipeline_path')
+def describe(json_option, pipeline_path):
+    """
+    Display pipeline description in a human readable format
+    """
+
+    _validate_pipeline_file_extension(pipeline_path)
+
+    pipeline_definition = \
+        _preprocess_pipeline(pipeline_path, runtime='local', runtime_config='local')
+
+    if not json_option:
+        click.echo("Version:")
+        for current_pipeline in pipeline_definition["pipelines"]:
+            click.echo("  " + str(current_pipeline["app_data"]["version"]))
+
+        has_description = False
+
+        for current_pipeline in pipeline_definition["pipelines"]:
+            if current_pipeline["app_data"]["properties"]["description"] != "":
+                if not has_description:
+                    click.echo("Description:")
+                    has_description = True
+                click.echo("  " + str(current_pipeline["app_data"]["properties"]["description"]))
+
+        if not has_description:
+            click.echo("No Description")
+
+        click.echo("Type:")
+        for current_pipeline in pipeline_definition["pipelines"]:
+            click.echo("  " + str(current_pipeline["app_data"]["properties"]["runtime"]))
+
+        has_dependencies = False
+
+        for current_pipeline in pipeline_definition["pipelines"]:
+            for node in current_pipeline["nodes"]:
+                if len(node["app_data"]["component_parameters"]["dependencies"]) >= 1:
```

Comment:
dependencies might not be there or be none, use get('dependencies')

Reply:
+1 to using `node["app_data"]["component_parameters"].get("dependencies")`.  I would also add that you really don't need to check that length is >= 1 since a zero-length dependency list won't get into the `for` loop anyway.  As a result, this `if` block could be replaced with:
```python
for dependency in node["app_data"]["component_parameters"].get("dependencies", []):
    click.echo("  " + str(dependency))
```